test/nls_testutils.jl

```diff
@@ -9,7 +9,10 @@
   end
   @testset "Check dimensions" begin
     check_nls_dimensions(lls)
-    check_nlp_dimensions(lls, exclude=[hess, hess_coord])
+    check_nlp_dimensions(lls, exclude = [hess, hess_coord])
+  end
+  @testset "Multiple precision support" begin
+    multiple_precision_nls(lls)
```

Comment:
I'm surprised this works. Doesn't it check the Jacobian of `F`? Am I forgetting something?

Reply:
Yes, it does but `jac_residual` is implemented. So, I am not sure why this is surprising

Reply:
Since `nls.Avals` always has the same type, I assume it wouldn't work. It didn't work for some other model, or maybe for this one in the past. What it currently does is `vals .= nls.Avals`, so `vals` retains it's type.